zk-keygen/Cargo.toml

```diff
@@ -0,0 +1,39 @@
+[package]
+name = "solana-zk-keygen"
+description = """
+Solana privacy-related key generation utility
+
+The tool currently supports two types of encryption keys that are used in the SPL Token-2022 program:
+  - ElGamal keypair that can be used for public key encryption
+  - AES128 key that can be used for an authenticated symmetric encryption (e.g. AES-GCM-SIV)
+"""
+publish = false
+version = { workspace = true }
+authors = { workspace = true }
+repository = { workspace = true }
+homepage = { workspace = true }
+license = { workspace = true }
+edition = { workspace = true }
+
+[dependencies]
+bs58 = { workspace = true }
+clap = { version = "3.1.5", features = ["cargo", "derive"] }
+dirs-next = { workspace = true }
+num_cpus = { workspace = true }
+solana-clap-v3-utils = { workspace = true }
+solana-cli-config = { workspace = true }
+solana-remote-wallet = { workspace = true, features = ["default"] }
+solana-sdk = { workspace = true }
+solana-version = { workspace = true }
+solana-zk-token-sdk = { workspace = true }
+tiny-bip39 = { workspace = true }
+
+[dev-dependencies]
+tempfile = "3.5.0"
```

Comment:
```suggestion
tempfile = { workspace = true }
```

Reply:
I think this should actually be changed in regular `keygen` as well... Sorry I missed it in a previously. I'll make the change.